jaeger-b3/README.md

```diff
@@ -1,16 +0,0 @@
-# jaeger-b3
-These components allow you to join traces with other Zipkin (B3)
-instrumented applications (such as Brave or zipkin-go-opentracing).
-
-# Usage
```

Comment:
This section will need to be moved to the jaeger-core readme.

Reply:
done